### PR TITLE
Fix: create /Users/$HOME/.arkade/bin always

### PR DESCRIPTION
**Pull request by guacamole**
Signed-off-by: guacamole <gunjanwalecha@gmail.com>

## Description

`arkade get <tool>` fails if `$HOME/.arkade/bin` doesn't exist. This PR tries to fix this issue by always creating the folder and ignores the error if any.

## Motivation and Context

Fix for #195 
- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?

Tested on MacOS
arkade-dir-fix

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

**Comment by guacamole** [comment]
@alexellis i just checked your comment on the issue, should I instead use the `InitUserDir()` and update this PR?

**Comment by tuananh** [comment]
@guacamole i think because of it's urgency, it's already fixed in this commit https://github.com/alexellis/arkade/commit/d538641dcc6895fe06c8843f60cb541dd13eb65b

**Comment by guacamole** [comment]
@tuananh sorry i missed it, i'll close this pr now :)

File: cmd/get.go [hunk 1]
```diff
@@ -65,6 +67,14 @@ and provides a fast and easy alternative to a package manager.`,
 			dlMode = get.DownloadArkadeDir
 		}
 
+		currentUser, err := user.Current()
```
Comment (alexellis) [inline cmd/get.go:70]: Today I learned about `os/user` 👍 thanks

Perhaps the existing code behind InitUserDir may benefit from using user.Current instead? Are there any drawbacks or reliances on syscalls/C?

**Pull request closed without merging**